sections/docker/bootstrap-using-node.md

```diff
@@ -0,0 +1,62 @@
+# Bootstrap container using node command instead of npm
+
+## One paragraph explainer
+
+We are used to see code examples where folks start their app using `CMD 'npm start'`. This is a bad practice. The `npm` binary will not forward signals to your app which prevents graceful shutdown (see #716). If you are using Child-processes they won’t be cleaned up correctly in case of unexpected shutdown, leaving zombie processes on your host. `npm start` also results in having an extra process for no benefit. To start you app use `CMD ['node','server.js']`.
```

Comment:
Running CMD [node...] will also result in zombies, no? Maybe the advice should be, when no child processes are used you may invoke 'node' directly. Whenever child processes are used, a real process manager should be used like 'TINY' @kevynb

Reply:
It seems to me that if you are not using child-processes, your fine with just running node as PID1 and handling signals. Anyway I added a sentence about it, because that's needed 👍